presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -446,6 +446,16 @@ public void testRowFieldAccessorInJoin()
                 "SELECT * FROM VALUES (11, 370, 1), (22, 781, 2), (33, 1234, 3)");
     }
 
+    @Test
+    public void testRowTypeWithReservedKeyword()
```

Comment:
same comment

Reply:
Agreed, but see the comment below:

```
 // This test is here because it only reproduces the issue (https://github.com/prestosql/presto/issues/1962)
// when running in distributed mode
```

Reply:
so it seems we  need dedicated test for planner (TestPlanner?), using distributed tpch runner.

Reply:
That's a great idea (for a separate change). For now, I'm moving it to TestTpchDistributedQueries with a comment.